### PR TITLE
feat: Add optional Tooltip to UserAvatar component

### PR DESCRIPTION
**Pull request by SimBig**
BREAKING CHANGE: `username` key in UserAvatar got renamed to 'initials'

### CODE REVIEW

File: src/Avatar/index.tsx [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
-    <Avatar
-      style={{ backgroundColor: color, color: MLL_THEME.backgroundColor }}
-    >
-      {props.username}
-    </Avatar>
+    <Tooltip title={props.toolTipLabel ?? null}>
+      <Avatar
+        style={{ backgroundColor: color, color: MLL_THEME.backgroundColor }}
+      >
+        {props.initials}
```
Comment (spawnia) [inline src/Avatar/index.tsx:21]: As this is the primarily displayed content, I would expect this:

```suggestion
        {props.children}
```

File: src/Avatar/index.tsx [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
+
+export function UserAvatar(props: AvatarProps & AvatarUserProps) {
+  const color = stringToHslaColor(props.initials);
   return (
-    <Avatar
-      style={{ backgroundColor: color, color: MLL_THEME.backgroundColor }}
-    >
-      {props.username}
-    </Avatar>
+    <Tooltip title={props.toolTipLabel ?? null}>
```
Comment (spawnia) [inline src/Avatar/index.tsx:17]: Needless complection, composition works fine instead. Especially given the Tooltip is optional.

**Review by spawnia — CHANGES_REQUESTED** [review]

File: src/Avatar/index.tsx (hunk 2 above)
Comment (SimBig) [reply src/Avatar/index.tsx:17]: that means you would prefer a new component that uses the label (UserAvatarWithLabel) and make the label property on this component obligatory?

Comment (spawnia) [reply src/Avatar/index.tsx:17]: Clients can simply do this as needed:

```tsx
<Tooltip title="Foo"><UserAvatar /></Tooltip>
```

No need to increase the complexity of this component or introduce a new one.

Comment (SimBig) [reply src/Avatar/index.tsx:17]: ok, I see

**Pull request closed without merging**